tensorflow/compiler/mlir/lite/ir/tfl_ops.cc

```diff
@@ -955,7 +956,7 @@ static LogicalResult Verify(ScatterNdOp op) {
     // Checks whether the last `(shape_type.getDimSize(0) - outermost_dim)`
     // dimensions of `updates` and `shape` are equal.
     for (auto shape_it : llvm::enumerate(shape_value)) {
-      auto i = shape_it.index();
+      long int i = shape_it.index();
```

Comment:
Unsure about this change. What is the warning here?

Reply:
Here auto seems to assign the objects being compared different types.
My attempt was to manually set the type of one of the two objects to resolve the warning.

I could use `int64_t` if `long int` is less appropriate.

Reply:
`int64_t` is better